### ISSUE TITLE
Scraper doesn't collect IG TV posts

### ISSUE THREAD

**Issue by paullovachy** [rarcega/instagram-scraper#453]
<img width="315" alt="Screen Shot 2019-10-14 at 4 34 17 PM" src="https://user-images.githubusercontent.com/28489897/66781268-d65b5a80-eea0-11e9-8d03-15112555f308.png">

This post is an example - scraper does not collect IG TV posts, just FYI - these will be missing from the meta data json.

**Comment by DilanBary** [comment]
yeah had same issue, found different scraper today https://github.com/drawrowfly/instagram-scraper it scrapes such posts

**Comment by archmord** [comment]
If you use android you can install quicksave to download igtv videos

**Comment by SMe12435** [comment]
> yeah had same issue, found different scraper today https://github.com/drawrowfly/instagram-scraper it scrapes such posts

How?? Couldn't scrape IGTV vids
